### PR TITLE
fuzz printf

### PR DESCRIPTION
**Pull request by sylvestre**

### CODE REVIEW

File: fuzz/fuzz_targets/fuzz_printf.rs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            CommandResult {
+                stdout: String::new(),
+                stderr: error_result.stderr,
+                exit_code: error_result.exit_code,
+            }
+        }
+    };
+
+    compare_result(
+        "expr",
```
Comment (cakebaker) [inline fuzz/fuzz_targets/fuzz_printf.rs:100]:
```suggestion
        "printf",
```

File: fuzz/fuzz_targets/fuzz_printf.rs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        "\\u0000",
+        "\\U00000000",
+        "%%",
+    ];
+    escape_sequences.choose(rng).unwrap().to_string()
+}
+
+fn generate_printf() -> String {
+    let mut rng = rand::thread_rng();
+    let format_specifiers = ["%s", "%d", "%f", "%x", "%o", "%c", "%b", "%q"];
```
[the remark this answers is not in the document]
Comment (sylvestre) [reply fuzz/fuzz_targets/fuzz_printf.rs:47]: yeah, it is just to start :)

**Review by cakebaker — COMMENTED** [review]
It looks fine to me for a first version, though I'm not sure whether we should merge it yet as @tertsdiepraam works on a rewrite of `printf`.

**Comment by sylvestre** [comment]
It doesn't matter because it is testing the binaries, it doesn't rely on internal :)

**Pull request merged**